release/artifacts.yaml

```diff
@@ -105,6 +105,8 @@
   type: jar-scala
 - target: //ledger/ledger-api-auth:ledger-api-auth
   type: jar-scala
+- target: //ledger/ledger-api-grpc-utils:ledger-api-grpc-utils
+  type: jar-scala
```

Comment:
this should also be published to maven central with `mavenUpload: True`.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3370/commits/33cba479dd607f180843c5a802ac7407380df801